### PR TITLE
More compliance reminders for onboarding checklist

### PR DESCRIPTION
**Pull request by brittag**
This is a draft to help address https://github.com/18F/cg-product/issues/321 - it needs some review and refinement, especially in the Atlas area. cc @mogul

### CODE REVIEW

**Comment by brittag** [comment]
Also cc @NoahKunin since this has some interaction with overall SSP work.

**Comment by brittag** [comment]
Notes:

* Make this page make a little more sense to new people, such as explaining team names / linking to explanations
* Remove vague/confusing/weird todo items

**Comment by afeld** [comment]
Sorry, just realizing you might not have wanted this merged yet 😁

File: OnboardingChecklist.md [hunk 1]
```diff
@@ -83,19 +87,25 @@ In order to get NewPerson productively contributing to the cloud.gov team, Buddy
...
-- [ ] Identify a straightforward, well-groomed first story, ideally something they could conceivably complete in their first two/three weeks using their existing skills. Discuss the context with them, then make them the assignee for the card.
-- [ ] Make suggestions for how your onboarding experience could have been improved as PRs on [the onboarding template](https://github.com/18F/cg-product/blob/master/OnboardingChecklist.md).
-- [ ] Confirm they have 2FA enabled for GitHub, and ask `#admins-github` to have them added to the [@18F/cloud.gov team](https://github.com/orgs/18F/teams/cloud-gov) on GitHub
+- [ ] Ask `#admins-github` to have them added to the [@18F/cloud.gov team](https://github.com/orgs/18F/teams/cloud-gov) on GitHub
 
-### Atlas-specific items
-- [ ] Ask `#admins-github` to have them added to the [@18F/cloud-gov-team](https://github.com/orgs/18F/teams/cloud-gov-ops) team on GitHub **(For contractors: Confirm they have cleared GSA security review before doing this one!)**
+### Atlas-specific required items
+
+- [ ] Help them review and understand the responsibilities of becoming a Cloud Operations team member, as listed in our SSP
```
Comment (afeld) [inline OnboardingChecklist.md]: We should try to move as much of this to our more...easily linkable...documentation as possible.

**Comment by brittag** [comment]
@afeld Oh yeah, I meant to address the notes before merging, but that's ok - it's an incremental improvement, and I'll just send another PR. :)

**Pull request merged**